openlibrary/macros/databarWork.html

```diff
@@ -36,7 +36,7 @@ <h3 class="header">
         $ edition = page.works and page
 
         $ render_times['databarWork: List widget'] = time()
-        $ lists_widget = render_template('lists/widget', edition or work, include_rating=True, include_header=False, include_widget=True, include_showcase=False, async_load=True)
+        $ lists_widget = render_template('lists/widget', edition or work, include_rating=True, include_header=False, include_widget=True, include_showcase=False, async_load=True, show_already_lists=True)
```

Comment:
show which of your lists this books is already on?

Maybe `show_my_active_lists`

Reply:
If you can't name 'em, delete 'em :)